Cargo.toml

```diff
@@ -1,57 +1,58 @@
 [workspace]
 members = [
-   "async_data_factory",
-   "async_db",
-   "async_ex1",
-   "async_ex2",
-   "async-graphql-demo",
-   "async_pg",
-   "awc_https",
-   "basics",
-   "casbin",
-   "cookie-auth",
-   "cookie-session",
-   "diesel",
-   "docker_sample",
-   "error_handling",
-   "form",
-   "hello-world",
-   "http-proxy",
-   "json",
-   "json_decode_error",
-   "json_error",
-   "jsonrpc",
-   "juniper",
-   "juniper-advanced",
-   "middleware",
-   "multipart",
-   "multipart-s3",
-   "multipart-async-std",
-   "openssl",
-   "protobuf",
-   "r2d2",
-   "redis",
-   "redis-session",
-   "run-in-thread",
-   "rustls",
-   "rustls-client-cert",
-   "shutdown-server",
-   "server-sent-events",
-   "simple-auth-server",
-   "state",
-   "static_index",
-   "template_askama",
-   "template_handlebars",
-   "template_tera",
-   "template_tinytemplate",
-   "template_yarte",
-   "todo",
-   "udp-echo",
-   "unix-socket",
-   "web-cors/backend",
-   "websocket",
-   "websocket-autobahn",
-   "websocket-chat",
-   "websocket-chat-broker",
-   "websocket-tcp-chat"
+   "async/data_factory",
+   "async/db",
+   "async/ex1",
+   "async/ex2",
+   "async/pg",
```

Comment:
async/db -> database/basic (or database/sqlite)
async/pg -> database/pg
async/ex1 -> basics/json-validation
async/ex2 -> basics/nested-routing

Reply:
What about async/data_factory? Move it to basics as well?

Reply:
basics or other, don't mind